src/Entities/Entity.cpp

```diff
@@ -943,6 +944,7 @@ void cEntity::HandlePhysics(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		{
 			// Normal gravity
 			fallspeed = m_Gravity * DtSec.count();
+			NextSpeed -= NextSpeed * m_AirDrag * DtSec.count();
```

Comment:
This calculation means that `m_AirDrag` is measured neither in "ratio per second" nor "ratio per tick".